frontend/packages/shared/src/components/altinnSubHeader/AltinnSubMenu.tsx

```diff
@@ -0,0 +1,21 @@
+import { BranchingIcon } from '@navikt/aksel-icons';
+import { ThreeDotsMenu } from 'app-development/layout/AppBar/ThreeDotsMenu';
+import { VersionControlHeader } from 'app-development/layout/version-control/VersionControlHeader';
+import classes from './AltinnSubMenu.module.css';
+import React from 'react';
+
+export const AltinnSubMenu = () => {
+  return (
+    <div data-testid='altinn-sub-menu'>
+      <div className={classes.subToolbar}>
+        <div className={classes.leftContent} data-testid='branching-icon'>
+          <BranchingIcon className={classes.branchIcon} />
+        </div>
+        <div className={classes.rightContent}>
+          <VersionControlHeader data-testid='version-control-header' />
```

Comment:
Same as for the other two internal header components. This is app-development specific. The contents of this component (i.e. where VersionControlHeader is now) should be passed in to the component so that it is generic.

Reply:
Added props